docs/guidance/build/merging.md

```diff
@@ -12,50 +12,38 @@ The following examples show how updates and deletions are reflected in compiled
 
 ## Example 1: Updates
 
-A public procurement agency publishes a release to announce an opportunity on January 1, 2016 in which the estimated value of the procurement is $1,000. On January 31, it publishes a release to expand the description of the procurement. On February 5, it publishes a release to amend the opportunity, in which the estimated value is increased to $2,000.
+A government agency in Ghana publishes a release to announce an opportunity on October 21, 2020 with an estimated value of 13,000 GHS. On November 5, 2020 they publish an update containing the details of 2 parties that have submitted bids and the publication date of the bidding documents.
```

Comment:
* Include the publisher's name
* 'they' -> 'it' to match 'publishes' rather than 'publish'
* '2' -> 'two' because it's usual in technical writing to write out numbers less than ten
* 'parties that have submitted bids' -> 'tenderers' for brevity, consistency with OCDS terminology and to avoid passive voice
```suggestion
The Ghana Public Procurement Authority publishes a release to announce an opportunity on October 21, 2020 with an estimated value of 13,000 GHS. On November 5, 2020 it publishes an update containing the details of two tenderers and the publication date of the bidding documents.
```